src/index.ts

```diff
@@ -602,3 +671,61 @@ export class Connection
         })
     }
 }
+
+function createPropFindBody(options : ConnectionReaddirOptions): [XMLElementBuilder, ConnectionReaddirProperty[]] {
+    const findProps = new XMLElementBuilder('d:propfind', {
```

Comment:
For multi-line arguments I would expect each argument to get its own line, even the first.